dfetch/project/metadata.py

```diff
@@ -99,6 +99,15 @@ def remote_url(self) -> str:
         """Branch as stored in the metadata."""
         return self._remote_url
 
+    @property
+    def last_fetch(self) -> datetime.datetime:
+        """Last fetch as stored in the metadata."""
+        return self._last_fetch
+
+    def last_fetch_string(self) -> str:
+        """Last fetch as stored in the metadata (converted to string)."""
+        return self.last_fetch.strftime("%d/%m/%Y, %H:%M:%S")
```

Comment:
This can use `self._last_fetch` directly, since that is a private member of this class.